prow/config.yaml

```diff
@@ -237,6 +237,45 @@ presubmits:
     rerun_command: "/test pull-kubernetes-cross"
     trigger: "(?m)^/test pull-kubernetes-cross,?(\\s+|$)"
     run_if_changed: '^(build\/|hack\/lib\/)|(Makefile)|(.*_(windows|linux|osx|unsupported)(_test)?\.go)$'
+  - name: pull-kubernetes-dind-build
+    agent: kubernetes
+    context: pull-kubernetes-dind-build
+    always_run: false
+    rerun_command: "/test pull-kubernetes-dind-build"
+    trigger: "(?m)^/test pull-kubernetes-dind-build,?(\\s+|$)"
+    spec:
+      containers:
+      - name: dind
+        image: docker:dind
+        securityContext:
+          priveleged: true
+        volumeMounts:
+        - name: docker-graph-storage
+          mountPath: /var/lib/docker
+      - name: build
+        image: gcr.io/k8s-testimages/e2e-prow
+        env:
+        - name: DOCKER_HOST
+          value: tcp://localhost:2375
+        args:
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --git-cache=/root/.cache/git
+        - --clean
+        - --timeout=110
```

Comment:
not sure about this setting, and the config in general, but I'm only aiming for a POC we can run from https://github.com/kubernetes/kubernetes/pull/46662